src/dag-link/index.js

```diff
@@ -16,25 +17,29 @@ class DAGLink {
     this._nameBuf = null
     this._size = size
     this._cid = new CID(cid)
+
+    // Make sure we have a nice public API that can be used by an IPLD resolver
+    visibility.hidePrivateFields(this)
+    visibility.addEnumerableGetters(this, ['Hash', 'Name', 'Tsize'])
```

Comment:
why are we going with capitalised names? is there something special about dag-pb that warrants this, otherwise it seems like it's straying into non-idiomatic-land.